internal/xpkg/name_test.go

```diff
@@ -73,6 +73,14 @@ func TestFriendlyID(t *testing.T) {
 			},
 			want: "provider-aws-plusabunchofothernonsensethatisgoingt-1234-5678912",
 		},
+		"DigestIsName": {
+			reason: "A valid DNS label should be returned when package digest is a name.",
+			args: args{
+				pkg:  "provider-in-cluster",
+				hash: "provider-in-cluster",
+			},
+			want: "provider-in-cluster-provider-in",
+		},
```

Comment:
presumably `ToDNSLabel` handles the replace `.`->`-` now?

Reply:
if so lgtm. fwiw if that was shown in the included test this would be an even more ez +1 👍

Reply:
@kasey test above this one already demonstrates :)

Reply:
Fair enough! LGTM